packages/nx/src/config/workspace-json-project-json.ts

```diff
@@ -111,6 +111,10 @@ export interface ProjectConfiguration {
       'generator' | 'generatorOptions'
     >;
   };
+  metadata?: {
+    technologies?: string[];
+    targetGroups?: Record<string, string[]>;
+  };
```

Comment:
Should we add this to the `project-schema.json`?

Also, did we consider additionally allowing any arbitrary metadata to be set? That could be useful for third-party plugins.

Reply:
I didn't add it to `project-schema.json` even though it works when defined in `project.json` because I'm not sure we should recommend it. At least for the metadata that we're going to consume, I think it should be mostly programmatic.

By the way, I believe we've actually always allowed for arbitrary properties on the project node.. Like it will end up on the project graph but we don't do anything with it.